R/write_dwc.R

```diff
@@ -138,18 +142,19 @@ write_dwc <- function(package, directory = ".", doi = package$id,
   }
 
   # Add extra paragraph to description
-  first_para <- glue::glue(
+  first_author <- eml$dataset$creator[[1]]$individualName$surName
+  pub_year <- substr(eml$dataset$pubDate, 1, 4)
+  first_para <- paste(
```

Comment:
Why do you switch from glue to paste here?

Reply:
I thought we needed to remove the dependency on `glue`, but I guess it was only `glue_sql()`?

Reply:
Oh right, I notice we only use it glue in 2 places (here and for `glue_sql`). Indeed better to remove then, including as dependency in https://github.com/inbo/movepub/blob/9279121a9d5b8beaa5e259ab418c64ad1dc1b2f7/DESCRIPTION#L26